pkg/console/operator/sync_v400.go

```diff
@@ -197,6 +215,17 @@ func (co *consoleOperator) sync_v400(updatedOperatorConfig *operatorv1.Console,
 
 func (co *consoleOperator) SyncConsoleConfig(consoleConfig *configv1.Console, consoleURL string) (*configv1.Console, error) {
 	updated := consoleConfig.DeepCopy()
+
+	// track the URL state in prometheus before we update it
```

Comment:
Making it clear if/when URL updates that we are no longer using the previous url.

```golang
// previous is no longer used
console_url{url="foobar"} 0
// new one in use
console_url{url="bazqux"} 1
```

Reply:
@brancz just to verify this is the correct way to use/consistent with how we are doing things elsewhere!